src/main/java/org/fundacionjala/at15/katas/langtonant/vladimir/Main.java

```diff
@@ -0,0 +1,86 @@
+package org.fundacionjala.at15.katas.langtonant.vladimir;
```

Comment:
For some reason, indentation was lost, which made me realize that we need to include indentation rule to checkstyle config.

Please add the following line to [checkstyle.xml](https://github.com/fjcoding/AT15/blob/dev/langtonant/config/checkstyle/checkstyle.xml) after line 176
```
<module name="Indentation"/>
```
Save the file and run `./gradlew check` again, it should guide you to fix indentation

Reply:
All ready done it!